packages/coinstac-ui/scripts/build.js

```diff
@@ -6,12 +6,14 @@ const rm = pify(require('rimraf'));
 const os = require('os');
 const path = require('path');
 
-const platform = ['linux', 'win32', 'darwin'];
```

Comment:
The equality checks for icon compare an array to strings using strict equality. _WAT._